### PR TITLE
Add test case in scheduler/profile/profile_test.go

### PR DESCRIPTION
**Pull request by KobayashiD27**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup

> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Add test case in scheduler/profile/profile_test.go to improve the test coverage: check invalid configs
The test case does not cover the error of invalid configs.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: pkg/scheduler/profile/profile_test.go [hunk 1]
```diff
@@ -273,6 +273,67 @@ func TestNewMap(t *testing.T) {
...
+						QueueSort: &config.PluginSet{
+							Enabled: []config.Plugin{
+								{Name: "QueueSort"},
+							},
+						},
+					},
+				},
+			},
+			wantErr: "at least one bind plugin is needed",
+		},
```
Comment (Huang-Wei) [inline pkg/scheduler/profile/profile_test.go:336]: it's duplicated with https://github.com/kubernetes/kubernetes/blob/5bda0c1b3b3b3f9cbfc65255d7258fe4b441410e/pkg/scheduler/profile/profile_test.go#L71-L84.

**Review by Huang-Wei — COMMENTED** [review]
Please move the cases to `TestNewProfile` as `TestNewMap` is focus on testing conflicts from multiple profiles.

File: pkg/scheduler/profile/profile_test.go (hunk 1 above)
Comment (KobayashiD27) [reply pkg/scheduler/profile/profile_test.go]: Thank you for your review.
I will remove it.

Comment (Huang-Wei) [reply pkg/scheduler/profile/profile_test.go]: Can you address this:

> Please move the cases to `TestNewProfile` as `TestNewMap` is focus on testing conflicts from multiple profiles.

Comment (KobayashiD27) [reply pkg/scheduler/profile/profile_test.go]: I missunderstood your comment.
However, `NewMap` does not call `validate`, so it does not occur an error in these invalid cases.
Could you tell me whether this behavior is ok or not?

Comment (Huang-Wei) [reply pkg/scheduler/profile/profile_test.go]: > However, NewMap does not call validate

I see, didn't notice that `NewProfile` doesn't check presence of schedulerName and plugins. Then let's just move the "one queue sort plugin required for profile" test to `NewProfile` - which is expected to throw an error.

Comment (KobayashiD27) [reply pkg/scheduler/profile/profile_test.go]: Thanks your review.
I will fix it.

**Pull request merged**